src/commands/sources/list.ts

```diff
@@ -3,26 +3,40 @@ import {Command} from '@oclif/command';
 import {
   Catalog,
   Dataset,
+  Distribution,
 } from '@netwerk-digitaal-erfgoed/network-of-terms-catalog';
 
+type DatasetDistribution = {
+  dataset: Dataset;
+  distribution: Distribution;
+};
+
 export class ListSourcesCommand extends Command {
   static description = 'List queryable sources';
 
-  protected render(datasets: readonly Dataset[]): void {
-    cli.table(datasets as Dataset[], {
+  protected render(distributions: readonly DatasetDistribution[]): void {
+    cli.table(distributions as DatasetDistribution[], {
       distributionTitle: {
         header: 'Source Name',
-        get: (dataset: Dataset) => dataset.name,
+        get: (distribution: DatasetDistribution) => distribution.dataset.name,
       },
       distributionId: {
-        header: 'Source ID',
-        get: (dataset: Dataset) => dataset.identifier,
+        header: 'Distribution URI',
```

Comment:
Or `Source URI` rather than `Distribution URI` (so as to hide that we use distributions internally)?